src/Components/WebAssembly/Build/src/Microsoft.AspNetCore.Components.WebAssembly.Build.csproj

```diff
@@ -8,6 +8,7 @@
     <IsShippingPackage>true</IsShippingPackage>
     <HasReferenceAssembly>false</HasReferenceAssembly>
     <GenerateDependencyFile>false</GenerateDependencyFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Are there any public facing APIs in this package? Maybe this one isn't needed.

Reply:
@pranavkm please let me know whether I need to revert this particular file or not.